### DISCUSSION TITLE
Event when user permissions are changed

### DISCUSSION THREAD

**Discussion by ryssbowh** [craftcms/cms#12130] [Ideas] [answered]
It would be good to be able to know when user permissions are changed, would it be possible to add an event in `UserPermissions::saveUserPermissions` ?

**Comment by brandonkelly** [comment, marked as answer]
We’ve added new `EVENT_AFTER_SAVE_GROUP_PERMISSIONS` and `EVENT_AFTER_SAVE_USER_PERMISSIONS` events for Craft 4.3 via #12146 🎉

**Reply by brandonkelly** [reply]
Craft 4.3 was released earlier this week with those events!
